src/derivkit/forecasting/integrations/getdist_fisher_samples.py

```diff
@@ -0,0 +1,213 @@
+    *,
+    names: Sequence[str] | None = None,
+    labels: Sequence[str] | None = None,
+    label: str = "Fisher (Gaussian)",
+    rcond: float | None = None,
+) -> GaussianND:
+    """Returns a GetDist ``GaussianND`` for the Fisher Gaussian.
+
+    Args:
+        theta0: Fiducial parameter vector with shape ``(p,)`` with ``p`` parameters.
+        fisher: Fisher matrix with shape ``(p, p)``.
+        names: Optional parameter names (length ``p``). Defaults to ``p0, p1, ...``.
+        labels: Optional parameter labels (length ``p``). Defaults to ``p_0, p_1, ...``.
+        label: Label attached to the returned object.
+        rcond: Cutoff passed to the Fisher (pseudo-)inverse when forming the covariance.
+
+    Returns:
+        A ``GaussianND`` with mean ``theta0`` and covariance from ``fisher``.
+
+    Raises:
+        ValueError: If shapes or names/labels lengths are inconsistent.
+
+    See Also:
+        ``getdist.gaussian_mixtures.GaussianND``.
+    """
+    theta0 = np.asarray(theta0, dtype=float)
+    fisher = np.asarray(fisher, dtype=float)
+    validate_fisher_shapes(theta0, fisher)
+
+    n_params = int(theta0.size)
+
+    default_names = [f"p{i}" for i in range(n_params)]
+    default_labels = [rf"p_{{{i}}}" for i in range(n_params)]
+
+    param_names = list(default_names if names is None else names)
+    param_labels = list(default_labels if labels is None else labels)
+
+    if len(param_names) != n_params or len(param_labels) != n_params:
+        raise ValueError("names/labels must match number of parameters")
+
+    covariance = fisher_to_cov(fisher, rcond=rcond)
+
+    return GaussianND(
+        mean=theta0,
+        cov=covariance,
+        names=param_names,
+        labels=param_labels,
+        label=label,
+    )
+
+
+def fisher_to_getdist_samples(
+    theta0: NDArray[np.floating],
+    fisher: NDArray[np.floating],
+    *,
+    names: Sequence[str],
+    labels: Sequence[str],
+    n_samples: int = 30_000,
+    seed: int | None = None,
+    kernel_scale: float = 1.0,
+    prior_terms: Sequence[tuple[str, dict[str, Any]] | dict[str, Any]] | None = None,
+    prior_bounds: Sequence[tuple[float | None, float | None]] | None = None,
+    logprior: Callable[[NDArray[np.floating]], float] | None = None,
+    hard_bounds: Sequence[tuple[float | None, float | None]] | None = None,
+    store_loglikes: bool = True,
+    label: str = "Fisher (samples)",
+) -> MCSamples:
+    """Draws samples from the Fisher Gaussian as GetDist ``MCSamples``.
+
+    Samples are drawn from a multivariate Gaussian with mean ``theta0`` and
+    covariance ``kernel_scale**2 * pinv(fisher)``. Optionally, samples are
+    truncated by hard bounds and/or by a prior (via ``logprior`` or
+    ``prior_terms``/``prior_bounds``).
+
+    GetDist stores “loglikes” as ``-log(posterior)`` up to an additive
+    constant. When ``store_loglikes=True`` we store:
+
+        ``-log p(theta) = 0.5 * (theta-theta0)^T F (theta-theta0) - logprior(theta) + const``.
```

Comment:
```suggestion
    constant. When ``store_loglikes=True`` ``fisher_to_getdist_samples`` stores::

        -log p(theta) = 0.5 * (theta-theta0)^T F (theta-theta0) - logprior(theta) + const
```
Please clarify the value of the constant.

Reply:
clarified in 364dc575868e7df76575739381537392a83e6d02